gdal/ogr/ogrct.cpp

```diff
@@ -926,6 +943,34 @@ OGRProjCT::OGRProjCT()
 {
 }
 
+/************************************************************************/
+/*                  OGRProjCT(const OGRProjCT& other)                   */
+/************************************************************************/
+
+//! @cond Doxygen_Suppress
+OGRProjCT::OGRProjCT(const OGRProjCT& other) :
+    poSRSSource((other.poSRSSource != nullptr) ? (other.poSRSSource->Clone()) : (nullptr)),
+    bSourceLatLong(other.bSourceLatLong),
+    bSourceWrap(other.bSourceWrap),
+    dfSourceWrapLong(other.dfSourceWrapLong),
+    poSRSTarget((other.poSRSTarget != nullptr) ? (other.poSRSTarget->Clone()) : (nullptr)),
+    bTargetLatLong(other.bTargetLatLong),
+    bTargetWrap(other.bTargetWrap),
+    dfTargetWrapLong(other.dfTargetWrapLong),
+    bWebMercatorToWGS84LongLat(other.bWebMercatorToWGS84LongLat),
+    nErrorCount(other.nErrorCount),
+    dfThreshold(other.dfThreshold),
+    m_pj((other.m_pj != nullptr) ? (proj_clone(OSRGetProjTLSContext(), other.m_pj)) : (nullptr)),
```

Comment:
as noted in OGRProjCT::GetInverse(), proj_clone might fail in some circumstances for PROJ < 8.0.1, so you need a fallback path that recomputes m_pj with part of the logic of OGRProjCT::Initialize(). In practice you might want to do that in OCTClone() by calling OGRCreateCoordinateTransformation()